docs/source/user_guide/first_steps.rst

```diff
@@ -29,97 +29,92 @@ When the application starts you should see this window::
 
 - a hierarchy of tasks to perform.
-- a set of tools which are most of them optional.
+- a set of tools which are mostly optional.
 
-The tasks are the true backbone of the measure. Each one describe an elementary
+The tasks are the true backbone of the measure. Each one describes an elementary
 step of the measure. They are organized in a tree structure hence allowing more
-flexibility than simply nested loops. Tasks can be pass information between 
+flexibility than simply nested loops. Information can be passed between Tasks  
 through a shared database.
 
-The tools are optional and allow to customize three parts of the execution:
+When creating a new blank measure (using the **File/New measure** menu), the panel 
+added to the graphical interface allows you to edit the tasks and the tools. 
+To edit the tools, you need to click the 
+**Edit tools** button ; the edition of tools will be described in ???.
 
-- pre-execution hooks are run before starting the actual measurement and can
-  be used to validate the parameters in the measure or collect the state of the
-  application.
-- monitors are active while the main part of the measure is running and can 
-  report on the progress of the measure.
-- post-execution hooks are run after the main part of the measure has been 
-  executed and can run even if the main part of measure errored.
-  
-When creating a new blank measure (using the **File/New measure** menu), the 
-panel added to the graphical interface allows you to edit the tasks, to edit
-the tools you need to click the **Edit tools** button. The edition of tools 
-will be described in ???.
-
-Each measure have a name, an id and a root directory. The name is intented to 
+Each measure has a name, an id and a root directory. The name is intented to 
 describe the purpose of the measure while its id can act as a counter to 
-discriminate multiple execution of the same measure. The root directory is used
-to save the measure and the log file associated with the measure and can also 
-be accessed by the tasks to serve as root directory to save the measure data.
-
-The task hierarchy starts empty with only a root task. From there you can use
-either the tree or the buttons in the right panel to add more tasks. The tree 
-view can also be used to re-organize the tasks and rename them. The panel on 
-the right contains the editors organized in tabs. By default the standard 
-editor is selected and allow to parametrize the tasks. For basic measure, this
+discriminate multiple execution of the same measure - it should be a unique 
+identifier. The root directory is use to save an '.ini' file corresponding to the 
+measure and the associated log file. It can also be accessed by the tasks to serve 
+as root directory to save the measured data.
+
+The task hierarchy starts empty, with only a (dummy) root task. From there you can use
+either the tree or the buttons in the right panel to add tasks. The tree 
```

Comment:
Should give more details here (context menu on the tree, possibility to drag and drop, > mark on the button in the standard editor)